packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -140,25 +183,42 @@ export class Coachmark extends BaseComponent<ICoachmarkProps, ICoachmarkState> {
   public componentDidMount(): void {
     // If we have already accessed the width and height
     // We do not wan't to do it again.
+    const entityInnerHostRect = this._entityInnerHostElement.offsetWidth;
+
     if (this.state.isMeasuring) {
       this._async.setTimeout((): void => {
+
         if ((this.state.entityInnerHostRect.width + this.state.entityInnerHostRect.width) === 0) {
+
+          // @TODO Eventually we need to add the various directions
+          const beakLeft = (this.props.width! / 2);
```

Comment:
This won't position the beak in the middle of the coachmark. This puts the leftmost part of the beak in the middle. So beakleft should be something like `this.props.width/2 - this.props.beakwidth/2`

Reply:
Yeah i actually take care of that in the styling by adding -50% to translateX but i could see how devs could get confused by this. I'm going to take your suggestion and remove the translateX bit in css so it's not confusing.